db/schema.rb

```diff
@@ -289,6 +289,7 @@
     t.text "custom_css"
     t.integer "pagination_type", default: 0
     t.integer "linked_id"
+    t.uuid "uuid", default: -> { "gen_random_uuid()" }
```

Comment:
I thought SQLite doesn't support UUIDs, or am I wrong?

Reply:
That is correct. I have PostgreSQL set up locally, so I didn't catch that snag.

Reply:
I've wanted to replace all IDs with UUIDs, but SQLite is so nice in dev, don't want to give it up